README.md

```diff
@@ -233,7 +233,7 @@ How to instantiate and use the Language Translation service:
 
 ```swift
 let service = LanguageTranslation(username: "yourusername", password: "yourpassword")
-service.getIdentifiableLanguages({(languages:[IdentifiableLanguage]?, error) in
+service.getIdentifiableLanguages({(languages:[LanguageTranslation.IdentifiableLanguage]?, error) in
```

Comment:
Please create the instance name such as languageTranslation. getIdentifiableLanguages